Copy Data DB support

SHAI-amit-kumar2 opened the pull request:
Added support for more DBs and REST based methods.

SHAI-amit-kumar2 committed:
Add universal copy data registration

SHAI-amit-kumar2 committed:
Add HTTP and REST source/sink support to universal copy activity tool

UtkarshSinghShorthillsAI commented on src/fabricmcp_server/tools/universal_copy_activity.py:
```diff
@@ -0,0 +1,940 @@
+# TOOL REGISTRATION
+# =============================================================================
+
+def register_universal_copy_tools(app: FastMCP):
```
make this function's definition consistent with the registration functions definition of existing tools. Rest Looks good to me.

SHAI-amit-kumar2 replied:
Changes have been made, please review them.

UtkarshSinghShorthillsAI reviewed (changes requested):
Looks good to me, Just make the registration function definition consistent with other tools registration function

SHAI-amit-kumar2 committed:
Make universal copy activity tool registration more robust and remove MongoDB commented code

- Changed tool registration from decorator approach to robust function call pattern
- Added proper logging following the pattern used by other modules
- Removed all MongoDB Atlas commented code blocks as requested
- Existing source/sink logic remains completely unaffected
- Improved code maintainability and consistency

SHAI-amit-kumar2 committed:
Add comprehensive source/sink support to universal copy activity tool

- Add FileSystem, MySQL, and Google Cloud Storage source/sink models
- Implement robust tool registration pattern following existing module standards
- Remove MongoDB commented code blocks for cleaner codebase
- Support all major data connector types with accurate JSON generation
- Preserve all existing source/sink logic and functionality
